llvm-hs-pure/src/LLVM/AST/Float.hs

```diff
@@ -2,19 +2,18 @@
 -- numbers LLVM supports. It is most definitely intended to be imported qualified.
 module LLVM.AST.Float where
 
-import Prelude as P
-import Data.Data
-import Data.Word (Word16, Word64)
+import LLVM.Prelude
+import LLVM.Prelude as P
```

Comment:
You should be able to remove this qualified importand just replace `P.Double` with `Double` below.